### ISSUE TITLE
Multiple possible root elements in XML

### ISSUE THREAD

**Issue by LogMANOriginal** [admin-shell-io/aas-specs#201]
The XML schema defines four possible root elements:

* `aasenv`
```xml
<?xml version="1.0" encoding="utf-8"?>
<aasenv xmlns="http://www.admin-shell.io/aas/3/0">
	<assetAdministrationShells></assetAdministrationShells>
	<assets></assets>
	<conceptDescriptions></conceptDescriptions>
	<submodels></submodels>
</aasenv>
```

* `key` (AAS.xsd)
```xml
<?xml version="1.0" encoding="utf-8"?>
<key xmlns="http://www.admin-shell.io/aas/3/0" />
```

* `key` (IEC61360.xsd)
```xml
<?xml version="1.0" encoding="utf-8"?>
<key xmlns="http://www.admin-shell.io/IEC61360/3/0" />
```

* `internalInformationPoints`
```xml
<?xml version="1.0" encoding="utf-8"?>
<internalInformationPoints xmlns="http://www.admin-shell.io/aas/abac/3/0" />
```

I believe that only `aasenv` is supposed to be a root element. If that is the case, then the following lines should be removed:

https://github.com/admin-shell-io/aas-specs/blob/d5d6768cd084d9ed41db6543ceaa442899a08ac0/schemas/xml/AAS.xsd#L505
https://github.com/admin-shell-io/aas-specs/blob/d5d6768cd084d9ed41db6543ceaa442899a08ac0/schemas/xml/IEC61360.xsd#L126
https://github.com/admin-shell-io/aas-specs/blob/d5d6768cd084d9ed41db6543ceaa442899a08ac0/schemas/xml/AAS_ABAC.xsd#L135

**Comment by mristin** [comment]
Hi @LogMANOriginal ,
Please have a look for the draft of 3.0RC2:
https://github.com/admin-shell-io/aas-specs/tree/draft-V3RC02-schemas/schemas/xml

I will also add data specifications and the security parts this Monday, since these parts are still pending.

**Comment by LogMANOriginal** [comment]
Thanks! I didn't notice the other branch. I'll wait for the data specifications and security parts.

Looks like this issue has been addressed already. If the other schemas don't add additional root elements, it can probably be closed.

**Comment by mristin** [comment]
Closing since fixed by #204.
